datafusion/core/tests/dataframe/dataframe_functions.rs

```diff
@@ -1310,8 +1310,8 @@ async fn test_count_wildcard() -> Result<()> {
         @r"
     Sort: count(*) ASC NULLS LAST [count(*):Int64]
       Projection: count(*) [count(*):Int64]
-        Aggregate: groupBy=[[test.b]], aggr=[[count(Int64(1)) AS count(*)]] [b:UInt32, count(*):Int64]
-          TableScan: test [a:UInt32, b:UInt32, c:UInt32]
+        Aggregate: groupBy=[[test.b]], aggr=[[count(Int64(1)) AS count(*)]] [test.b:UInt32, count(*):Int64]
+          TableScan: test [test.a:UInt32, test.b:UInt32, test.c:UInt32]
```

Comment:
Does the `test` already tell us the qualifier?

Reply:
> More generally, when there is only one relation in the query, as is the case in many queries, adding a qualifier to all expressions I think makes the plans harder to read, not better

Agreed.
But also, single-table queries are not the ones we should optimize EXPLAIN output for.
These represent a subset of all queries which naturally is simpler than all queries, without source table count limit.